msfvenom

```diff
@@ -347,17 +399,20 @@ if generator_opts[:list_options]
 
   if payload_mod.nil?
     $stderr.puts "Invalid payload: #{generator_opts[:payload]}"
-    exit
+    exit(1)
   end
 
-  $stderr.puts "Options for #{payload_mod.fullname}:\n\n"
+  $stderr.puts "Options for #{payload_mod.fullname}:\n" + "="*25 + "\n\n"
   $stdout.puts ::Msf::Serializer::ReadableText.dump_module(payload_mod, '    ')
 
-  $stderr.puts "Advanced options for #{payload_mod.fullname}:\n\n"
+  $stderr.puts "\nAdvanced options for #{payload_mod.fullname}:\n" + "="*25 + "\n\n"
   $stdout.puts ::Msf::Serializer::ReadableText.dump_advanced_options(payload_mod, '    ')
 
-  $stderr.puts "Encryption options for #{payload_mod.fullname}:\n\n"
-  $stdout.puts ::Msf::Serializer::ReadableText.dump_encrypt_options(payload_mod, '    ')
+  $stderr.puts "\nEvasion options for #{payload_mod.fullname}:\n" + "="*25 + "\n\n"
+  $stdout.puts ::Msf::Serializer::ReadableText.dump_evasion_options(payload_mod, '    ')
+
+  #$stderr.puts "\nEncryption options for #{payload_mod.fullname}:\n" + "="*25 + "\n\n"
+  #$stdout.puts ::Msf::Serializer::ReadableText.dump_encrypt_options(payload_mod, '    ')
```

Comment:
`dump_encrypt_options` isn't in use anywhere?

Reply:
Looks like this was just incorrectly converted from `evasion` to `encrypt` in #9869 likely due to some of the refactoring that happened on that PR.

You have corrected our error. 👍